examples/building_stock_weather_aggregation.ipynb.license

```diff
@@ -1,3 +1,3 @@
-SPDX-FileCopyrightText: 2021 - 2023 Topi Rasku
+SPDX-FileCopyrightText: 2021 - 2024 Topi Rasku
```

Comment:
```suggestion
SPDX-FileCopyrightText: 2023 Topi Rasku
```

The last substantial update warranting a (c) year refresh was 2023, which was also the first commit for this file. We should just remove the year. If we don't remove it, the only correct year is 2023 for this file

Reply:
Is this the case? Tests right now run on `pip`/`uv`